tests/common/reboot.py

```diff
@@ -493,3 +504,44 @@ def check_reboot_cause_history(dut, reboot_type_history_queue):
     logger.error("The number of expected reboot-cause:{} is more than that of actual reboot-cuase:{}".format(
         reboot_type_history_len, len(reboot_type_history_queue)))
     return False
+
+
+def try_create_dut_console(duthost, localhost, conn_graph_facts, creds):
```

Comment:
@JibinBao How are you ensuring the reboot command is issued via console connection? What if the reboot is issued via SSH connection and there is no console connection?

Reply:
The function is to create dut_console to collect conosle logs, and it is not related to reboot test logic.
If there is no console connection, it will not collect the console logs, and it does not affect the original reboot test.
It is an enhancement.  If there is a console connection and the console connection can be created successfully, it collect the console logs, otherwise, it will do nothing.